JamfUploaderProcessors/JamfUploaderLib/JamfUploaderBase.py

```diff
@@ -462,17 +462,21 @@ def get_jamf_pro_version(self, jamf_url, token):
                 jamf_pro_version = str(r.output["version"])
                 self.output(f"Jamf Pro Version: {jamf_pro_version}")
                 return jamf_pro_version
-            except (KeyError, AttributeError) as error:
+            except KeyError as error:
                 self.output(f"ERROR: No version received.  Error:\n{error}")
-                raise ProcessorError("Unable to determine version of Jamf Pro") from error
+                raise ProcessorError("No version received") from error
```

Comment:
I would prefer `"No version of Jamf Pro received"`

Reply:
I've made this requested change in ee70c26.